github/actions_workflow_runs_test.go

```diff
@@ -464,6 +464,7 @@ func TestWorkflowRun_Marshal(t *testing.T) {
 		HeadBranch: String("hb"),
 		HeadSHA:    String("hs"),
 		RunNumber:  Int(1),
+		RunAttempt: Int(1),
```

Comment:
Please revert this file... this somehow got merged in from another PR and should not be in this one.